index.js

```diff
@@ -259,14 +268,16 @@ module.exports = function(THREE) {
                 // restrict phi to be betwee EPS and PI-EPS
                 phi = Math.max( EPS, Math.min( Math.PI - EPS, phi ) );
 
+                scale += scaleDelta * (( this.enableDamping === true ) ? this.dampingFactor : 1);
```

Comment:
I imagined that only one scale-related property is needed. Having 2 is confusing. Have doubts whether the zoom changes as expected and would still work if radius was not limited by min and max distances